azure-app-service/massdriver.yaml

```diff
@@ -0,0 +1,181 @@
+schema: draft-07
+name: <md .Name md>
+description: <md .Description md>
+source_url: github.com/YOUR_ORG/<md .Name md>
+access: private
+type: application
+
+app:
+  envs: {}
+    # Use jq expressions to build environment variables from input params or connections
+    # LOG_LEVEL: .params.log_level
+    # MONGO_DSN: .connections.mongo_authentication.data.authentication.username + ":" + .connections.mongo_authentication.data.authentication.password + "@" + .connections.mongo_authentication.data.authentication.hostname + ":" + (.connections.mongo_authentication.data.authentication.port|tostring)
+  policies: []
+    # Use jq expressions to select policies from artifact security blocks
+    # - .connections.sqs.data.security.iam.subscribe
+
+params:
+  examples:
+    - __name: Development
+      application:
+        sku_name: S1
+        minimum_worker_count: 1
+        maximum_worker_count: 3
+    - __name: Production
+      application:
+        sku_name: P3v3
+        minimum_worker_count: 1
+        maximum_worker_count: 10
```

Comment:
I had a similar conversation w/ beebe on cloud run. 10 feels really low for a production service that may need to scale. We don't want people to look at this guided config and think "massdriver only supports toy-scale apps" app service default max workers is 30 I say we at least follow that.

From azure best practices:
>The default limit of App Service workers is 30. If the App Service routinely uses 15 or more instances, consider opening a support ticket to increase the maximum number of workers to 2x the instance count required to serve normal peak load.

Reply:
https://docs.microsoft.com/en-us/azure/architecture/framework/services/compute/azure-app-service/operational-excellence